src/main/java/com/pingcap/tikv/meta/IndexType.java

```diff
@@ -18,8 +18,9 @@
 import com.pingcap.tikv.exception.TiClientInternalException;
 
 public enum IndexType {
```

Comment:
toString method kinda tells us 0 is invalid, 1 is BTREE and 2 is HASH. Is it better we add some explanations here?

Reply:
Like BTREE-Fake?

Reply:
Actually the enum here for now is quite meaningless. Since we are not either using btree or normal hash index. It's added for kind compatibility.

Reply:
I've added comments on the top.